src/Donations/PaymentMethods/NewPaypal.tsx

```diff
@@ -67,11 +67,28 @@ function NewPaypal({
   }
 
   const onError = (data) => {
-    setPaymentError(`Your order ${data.orderID} failed due to some error.`);
+    setPaymentError(`Your order failed due to some error.`);
+
+    // This function shows a transaction success message to your buyer.
+    data = {
+      ...data,
+      type: "sdk",
+      status: "error",
+      errorMessage: data?.message,
+    };
+    payDonationFunction("paypal", "paypal", data);
   };
 
-  const onCancel = (data) => {
+  const onCancel = (data, actions) => {
     setPaymentError("Order was cancelled, please try again");
+
+    // This function shows a transaction success message to your buyer.
```

Comment:
Why does the comment is talking about a `transaction success message` if something got cancelled?